AmericanMedicalAssociation/AMA-style-guide
# Ewl 3741 topic member pitch

## Ticket(s)

**Jira Ticket**

- [EWL-3741: Topics | SG | Design Member Pitch CTA](https://issues.ama-assn.org/browse/EWL-3741)

## Description

Refactors atom list unordered to accept json data instead of hard-coded placeholder text
Creates molecule block-cta-with-list.
Creates organism topic-rail-right containing block-cta-with-list
Adds organism topic-rail-right to template topic

## To Test

- [x] Go to Atoms > Lists > Unordered
- [x] Go to Molecule > Blocks > block-cta-with-list
- [x] Go to Organisms > Topic > topic-rail-right
- [x] Go to Templates > Topic > Topic Hub Template

Observe all the patterns look sensible and approximate the MVP from the wireframes. 
Review markup for each and see that it, too, looks sensible.


## Relevant Screenshots/GIFs
Wireframe:

<img width="266" alt="screen shot 2017-07-27 at 4 34 12 pm" src="https://user-images.githubusercontent.com/28711067/28693308-a3f43736-72e9-11e7-827e-dbb544736c51.png">


SG:
![screen shot 2017-07-27 at 4 29 37 pm](https://user-images.githubusercontent.com/28711067/28693300-94b31968-72e9-11e7-89e0-bdeec5a60e99.png)

Wireframe:

![screen shot 2017-07-27 at 4 30 50 pm](https://user-images.githubusercontent.com/28711067/28693322-afc94510-72e9-11e7-9fc8-dfa9a0ce5e3d.png)

SG:
![screen shot 2017-07-27 at 4 29 04 pm](https://user-images.githubusercontent.com/28711067/28693290-8b905350-72e9-11e7-82c2-d704bf0d1ab4.png)



## Remaining Tasks

Since we're handling things a bit more iteratively this cycle, there may be questions/imperfections that remain and that we'll catch as we build out more. Those things will be entered into new tickets.


## Additional Notes

The rail right doesn't attach to the right because it's basically waiting for other columns to come before it. I thought about floating it, but we don't really use floats much so I left it as is even though it might for the moment appear slightly confusing.
